src/css/editor.css.js

```diff
@@ -1,4 +1,4 @@
-module.exports = `/*
+/*
```

Comment:
is this comment needed? maybe we could remove it as well?

Reply:
this is useful for generating the css code below it